src/localization/resources.ts

```diff
@@ -0,0 +1,13 @@
+import components from '@/public/locales/en/components.json';
+
+/**
+ * This object in combination with `i18next.d.ts` enables intellisense and syntax
+ * highlighting when accessing the translation units, e.g., useTranslation('components');
+ */
+export const resources = {
+  en: {
+    components: components
+  }
```

Comment:
should this be `en-US` ? remembering Alex's comment

Reply:
Yeees.. `en-US` please 🙏      ISO 639-1 + 3166-1 ftw